src/Utopia/Messaging/Adapter/Email/Mailgun.php

```diff
@@ -55,13 +55,21 @@ protected function process(EmailMessage $message): string
 
         if (! \is_null($message->getCC())) {
             foreach ($message->getCC() as $cc) {
-                $body['cc'] = "{$body['cc']},{$cc['name']}<{$cc['email']}>";
+                if (! empty($cc['name'])) {
+                    $body['cc'] = "{$body['cc']},{$cc['name']}<{$cc['email']}>";
+                } else {
+                    $body['cc'] = "{$body['cc']}, <{$cc['email']}>";
```

Comment:
Does it matter if the angle brackets are there with no name? Should it just be `"{$body['cc']}, {$cc['email']}"` instead?